apps/finance/app/src/components/AppLayout.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { AppBar, AppView, Button, ButtonIcon, Viewport, font } from '@aragon/ui'
+import MenuButton from './MenuButton/MenuButton'
+
+const AppLayout = ({
+  children,
+  title,
+  afterTitle,
+  smallViewPadding,
+  onMenuOpen,
+  mainButton,
+}) => {
+  return (
+    <Viewport>
+      {({ below }) => (
+        <AppView
+          padding={below('medium') ? smallViewPadding : 30}
+          appBar={
+            <AppBar>
+              <AppBarContainer
+                style={{ padding: below('medium') ? '0' : '0 30px' }}
+              >
+                <Title>
+                  {below('medium') && <MenuButton onClick={onMenuOpen} />}
+                  <TitleLabel>{title}</TitleLabel>
```

Comment:
How could this work for an app like `Permissions` that has a `NavigationBar`?

Reply:
This component is only meant to be used on apps that don’t use `NavigationBar` for now. I’d like to think a bit about `AppView`, `NavigationBar` and this component, to see how we can make the API a bit nicer for users (and us).